source/optional_features/custom_branding/index.html.md.erb

```diff
@@ -19,6 +19,12 @@ customisation for the following features:
 * the background and border colour of the top banner
 * the branding of your payment confirmation email
 
+You must tell us which service you want to apply custom branding to. Custom branding applies to the test and live environments for a service.
+
+We recommend that you only enable custom branding once you have [gone live and started taking payments from real users](/switching_to_live/#go-live).
```

Comment:
I think we sort of recommend this as it's easier for us, not necessarily because it's better for them or their testing/services. How do we handle that?

Reply:
well, we could just omit the statement entirely, or keep it in unchanged and take the benefit :)

Reply:
agree to remove